entryDetail.go

```diff
@@ -400,29 +400,39 @@ func (ed *EntryDetail) SetReceivingCompany(s string) {
 	ed.IndividualName = s
 }
 
-// SetCTXAddendaRecords setter for CTX AddendaRecords characters 1-4 of underlying IndividualName
-func (ed *EntryDetail) SetCTXAddendaRecords(i int) {
+// OriginalTraceNumberField is used in ACK and ATX files but returns the underlying IdentificationNumber field
+func (ed *EntryDetail) OriginalTraceNumberField() string {
+	return ed.IdentificationNumberField()
+}
+
+// SetOriginalTraceNumber setter for ACK and ATX OriginalTraceNumber which is underlying IdentificationNumber
+func (ed *EntryDetail) SetOriginalTraceNumber(s string) {
+	ed.IdentificationNumber = s
+}
+
+// SetCATXAddendaRecords setter for CTX and ATX AddendaRecords characters 1-4 of underlying IndividualName
+func (ed *EntryDetail) SetCATXAddendaRecords(i int) {
 	ed.IndividualName = ed.numericField(i, 4)
 }
 
-// SetCTXReceivingCompany setter for CTX ReceivingCompany characters 5-20 underlying IndividualName
+// SetCATXReceivingCompany setter for CTX and ATX ReceivingCompany characters 5-20 underlying IndividualName
 // Position 21-22 of underlying Individual Name are reserved blank space for CTX "  "
-func (ed *EntryDetail) SetCTXReceivingCompany(s string) {
+func (ed *EntryDetail) SetCATXReceivingCompany(s string) {
```

Comment:
This will break any users who had the old method, is that okay?

Reply:
That's correct, that would have to update the methods to use the new methods.   I can leave the old methods and we can deprecate them later.   I'm not sure how many clients are using the CTX* functions.  This may be a good way to find out.  Let me know if you want me to leave the CTX methods and deprecate later.

Reply:
Can we ask on the mailing list? cc @wadearnold

Reply:
Sure, the company the updates would know right away.  It's not like they would update production.  I'll send out a note to the mailing list.

Reply:
We can ask the mailing list. I don't understand the motivation for the renaming of CTX -> CATX ?

Reply:
To be a bit more relevant to what it supports both CTX and ATX EntryDetail, although I don't really like it.

I can leave CTX and both CTX EntryDetail and ATX EntryDetail can use it.  I don't like that either.

We can create ATX specific functions.  In this case specific to adding it in EntryDetail, I don't like that either.

So I chose the one I thought was best from 3 solutions I didn't like.  If it's agreed that another above is more preferable or some other solution is better I can make the adjustment.  Just let me know.